server/pages/index.js

```diff
@@ -180,8 +180,7 @@ function getDefaultContext( request ) {
 		// use ipv4 address when is ipv4 mapped address
 		clientIp: request.ip ? request.ip.replace( '::ffff:', '' ) : request.ip,
 		isDebug: context.env === 'development' || context.isDebug,
-		tinymceWpSkin: context.urls[ 'tinymce/skins/wordpress/wp-content.css' ],
-		tinymceEditorCss: context.urls[ 'editor.css' ]
+		urls: context.urls,
```

Comment:
`context.urls` is filled with the urls of every generated chunks. It's probably not required for now